projects/1.20/assets/raxiores-dinosaur-mode/raxiores/lang/zh_cn.json

```
@@ -0,0 +1,118 @@
+{
+  "This chest locked.": "这个箱子已上锁。",
+  "subtitles.giganoto.roar": "南方巨兽龙咆哮",
+  "subtitles.rex.bite": "霸王龙撕咬",
+  "gui.raxiores.call_members.button_write_name_here": "点击召唤",
+  "effect.raxioresold.frightened": "恐惧",
+  "item.raxiores.ladro_stocker_spawn_egg": "拉多斯托克生成蛋",

```

```suggestion
  "item.raxiores.ladro_stocker_spawn_egg": "拉多斯托克刷怪蛋",
```
原版的固有名词应与Minecraft原版一致，如有特殊用法请指出。
具体可见我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **4. 专有名词** 一节。
你也可以通过上方评论中，bot生成的对比（diff）文件或 [Minecraft 译名标准化](https://zh.minecraft.wiki/w/Minecraft_Wiki:%E8%AF%91%E5%90%8D%E6%A0%87%E5%87%86%E5%8C%96) 以获取更多的原版翻译帮助。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。